### ISSUE TITLE
OTFeatureFreezer without ligatures

### ISSUE THREAD

**Issue by nashpatty** [tonsky/FiraCode#1374]
Is it possible to disable ligatures and use OTFeatureFreezer to build the font without ligatures? I can't find the _code_ for it anywhere on the repo, if such a feature exists.
I could use Fira Mono, but I like the increase in size of some of the characters, particularly the + and * symbols in Fira Code.

**Comment by tonsky** [comment]
I think some people have used it, as far as I understand, it works for some features, but not for others

**Comment by tonsky** [comment]
Sorry I didn’t read that thoroughly. People use it to enable stylistic sets, not sure about turning off ligatures. You might try disabling `calt` to disable ligatures

**Comment by VladWinner** [comment]
> I think some people have used it, as far as I understand, it works for some features, but not for others

Yes. https://github.com/twardoch/fonttools-opentype-feature-freezer/issues/20#issuecomment-650642262 
Due to the lack of binding to Unicode, OTFeatureFreezer cannot completely bake (enable) `cv03-cv06`, zero (`cv11-cv13`), `cv14`, `onum`, `ss04`, `cv18`, `ss03`, `cv15-cv16`, `cv29`.
e.g. `Cannot remap 'numbersign_braceleft.liga' -> 'numbersign_braceleft.liga.cv29' because neither has a Unicode value assigned in any of the cmap tables.`

> not sure about turning off ligatures

Yes, this software is not designed to disable ligatures.

**Comment by gmr458** [comment]
is there an alternative to that software that can disable ligatures?

**Comment by brzzdev** [comment]
@tonsky Is there any update on the configurator?
